### PR TITLE
Make user management table responsive

### PR DESCRIPTION
**Pull request by louis-lau**
Now I'm not sure where you want the css for this, I just put it at the bottom of base.scss. Tell me if you want it somewhere else 👍 

https://vgy.me/Wec0UI.gif

### CODE REVIEW

File: src/Ombi/ClientApp/styles/base.scss [hunk 1]
```diff
@@ -858,4 +858,70 @@ a > h4:hover {
...
+            top: 6px;
+            left: 6px;
+            width: 45%;
+            padding-right: 10px;
+            white-space: nowrap;
+        }
+    }
+    /* Label the data */
+    .td-checkbox:before {
+        content: "Select:";
```
Comment (tidusjar) [inline src/Ombi/ClientApp/styles/base.scss]: We shouldn't be doing this in the CSS for localization reasons, same with the below

**Comment by tidusjar** [comment]
Nice job! But please see comments 😄

**Comment by louis-lau** [comment]
Better? Better.

File: src/Ombi/ClientApp/styles/base.scss [hunk 2]
```diff
@@ -895,33 +896,9 @@ a > h4:hover {
...
-
-    .td-last-logged-in:before {
-        content: "Last Logged In:";
-    }
-
-    .td-user-type:before {
-        content: "User Type:";
+        /* Label the data */
+        .td-labelled:before {
+            content: attr(data-label)
```
Comment (tidusjar) [inline src/Ombi/ClientApp/styles/base.scss]: I didn't know you could do that!

**Comment by tidusjar** [comment]
Thanks!

File: src/Ombi/ClientApp/styles/base.scss (hunk 2 above)
Comment (louis-lau) [reply src/Ombi/ClientApp/styles/base.scss]: Me neither!

**Pull request merged**